src/uwtools/api/sfc_climo_gen.py

```diff
@@ -0,0 +1,42 @@
+"""
+API access to the uwtools sfc_climo_gen driver.
+"""
+from pathlib import Path
+from typing import Dict
+
+import iotaa
+
+from uwtools.drivers.sfc_climo_gen import SfcClimoGen
+
+
+def execute(
+    task: str,
+    config_file: Path,
+    batch: bool = False,
+    dry_run: bool = False,
+) -> bool:
+    """
+    Execute an sfc_climo_gen task.
```

Comment:
SRW documentation refers to the `make_sfc_climo` task, while `sfc_climo_gen` refers to the executable. I see we have the `chgres_cube` driver coming next, which is an executable created by the build process as well. For future drivers, should we only refer to the executable name?

Reply:
I think that _drivers_ should use the name of the executable. Well, I mean, you can technically name an executable program whatever you want, and there's not necessarily a canonical name for it. I've seen, for example, FV3 compiled to an executable program called just `fv3`, or `fv3.exe`, or `ufs_model`. But calling it "FV3" / `fv3` seems sensible to me. Hard to say about `sfc_climo_gen`: The [Fortran source code](https://ufs-community.github.io/UFS_UTILS/sfc_climo_gen/driver_8F90_source.html) says that the `program` is called `driver`, which is pretty generic. So, here, I feel like a community-recognized name for the program itself is probably the right identifier to use for the driver.

If there are other actions that need to be taken to "make surface climatology" (which I guess is what the name `make_sfc_climo` is getting at, then I can imagine a higher-level task called `make_sfc_climo` that would have a constituent task that runs the `sfc_climo_gen` component via the driver implemented in this PR, but possibly take other actions either based on the `sfc_climo_gen` executable's output or other actions.

So, I guess I'm thinking that `sfc_climo_gen` is an appropriate name for a _component driver_, and `make_sfc_climo` might be an appropriate name for a TDB _experiment task_ that does additional work beyond running this driver. I hope that makes sense, and I'm happy to hear other ideas if it sounds wrong!